src/addrman.h

```diff
@@ -554,6 +571,28 @@ class CAddrMan
         Check();
     }
 
+    //! See if any to-be-evicted tried table entries have been tested and if so resolve the collisions.
+    void ResolveCollisions(){
```

Comment:
Nit: space after `()` before `{`. I think functions actually put the `{` on the next line, which you did in other cases.

Reply:
I just double checked the style guide and you are correct current style is `{` on a new line for methods.